packages/mattermost-redux/src/types/teams.ts

```diff
@@ -52,10 +52,23 @@ export type TeamsState = {
 
 export type TeamUnread = {
     team_id: string;
+
+    // The number of unread mentions in channels on this team, not including DMs and GMs
     mention_count: number;
-    msg_count: number;
+
+    // The number of unread mentions in root posts in channels on this team, not including DMs and GMs
     mention_count_root: number;
+
+    // The number of unread posts in channels on this team, not including DMs and GMs
+    //
+    // Note that this differs from ChannelMember and ChannelUnread since it tracks unread posts instead of read posts.
+    msg_count: number;
```

Comment:
While not strictly necessary, the `/** */` syntax can attach to the type of following member for better discovery.

<img width="538" alt="image" src="https://user-images.githubusercontent.com/11724372/131690858-34547471-4928-43f3-92d0-de0780ee0546.png">

```suggestion
    /**
     * The number of unread posts in channels on this team, not including DMs and GMs
     * @remarks Note that this differs from ChannelMember and ChannelUnread since it tracks unread posts instead of read posts.
     */
    msg_count: number;
```

Reply:
Oh interesting. I didn't know there was special syntax to trigger that. I had wondered why that shows up sometimes and not other times.

Reply:
Comments updated. I wasn't able to use `@link` there since the files don't reference each other, but I did use them in some of the other ones.